dropwizard-logging/src/main/java/io/dropwizard/logging/AbstractAppenderFactory.java

```diff
@@ -149,6 +151,19 @@ public TimeZone getTimeZone() {
     }
 
     @JsonProperty
+    public void setTimeZone(String zoneId) {
+        if (zoneId.equalsIgnoreCase("system")) {
+            this.timeZone = TimeZone.getDefault();
+        } else {
+            if (Arrays.asList(TimeZone.getAvailableIDs()).contains(zoneId)) {
+                this.timeZone = TimeZone.getTimeZone(zoneId);
+            } else {
+                LoggerFactory.getLogger(getClass()).warn("Unknown timezone '{}' provided, using UTC", zoneId);
```

Comment:
Right, it breaks with a time zone in the custom format (like `GMT-02:00`). Let's then remove the check for available time zones and just use `TimeZone.getTimeZone` as Jackson if the zone id is not `system`.